packages/nocodb/src/db/BaseModelSqlv2.ts

```diff
@@ -5013,6 +5013,29 @@ export function extractCondition(
       [logicOp, alias, op, value] =
         str.match(/(?:~(and|or|not))?\((.*?),(\w+)\)/)?.slice(1) || [];
     }
+
+    // handle isblank and isnotblank filter format
+    switch (op) {
+      case 'is':
+        if (value === 'blank') {
+          op = 'blank';
```

Comment:
`(field,is,blank)` : what will happen if field actually had value `blank` and we wanted to filter it? we should not support this case IMO. `field,isblank` `field,isnotblank` should suffice

Reply:
Actually for that `eq` is what user should have used

Reply:
Ah, my bad. Got it